services/bots/src/discord/commands/message.ts

```diff
@@ -76,7 +83,7 @@ export class MessageCommand implements DiscordTransformedCommand<MessageDto> {
     await interaction.reply({
       embeds: [
         new EmbedBuilder({
-          description: this.messageData[messageKey].content,
+          description: [userMention, this.messageData[messageKey].content].join(' '),
```

Comment:
Should this be stripped?

```
> [undefined, 'bla'].join(' ')
< " bla" 
```

Reply:
I thought it was needed at first, but it seems Discord does that automagically
This is the result without a mention;
`Hello world :wave:`